box2d/src/test/kotlin/ktx/box2d/worldsTest.kt

```diff
@@ -171,4 +171,44 @@ class WorldsTest : Box2DTest() {
     assertSame(body.fixtureList[0], callbackParameter)
     world.dispose()
   }
+
+  @Test
+  fun `should ray-cast between two vectors`() {
+    val world = createWorld()
+
+    val expectedEdge = world.body {}.edge(from = Vector2.Zero, to = Vector2(0f, 2f)) {}
```

Comment:
I think readability of this declaration can be improved with:

```Kotlin
    world.body {
      edge(from = Vector2.Zero, to = Vector2(0f, 2f)) {}
    }
```

Don't worry too much about this not being a one-liner. Same thing about the other test.